minvws/nl-kat-coordination
# feat: Updated color scheme

### Changes
- Adds updated color scheme and updates references

### Issue link

Closes:
https://github.com/minvws/nl-kat-coordination/issues/3163

### QA notes

- Please check for colors that are off/weird looking or missing. As the references have changed this could happen.
- In the scss check for any copy/paste mistakes

---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
